packages/components/molecules/f-rating/src/components/RatingFiveStarVariant.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div
+        :class="$style['c-rating']"
+        data-test-id="rating-component">
+        <div :class="$style['c-rating-stars-icons']">
+            <star-icon
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="[
+                    $style['c-rating-star'],
+                    $style['c-rating-star--empty'],
+                    $style[`c-rating-star--${starRatingSize}`]
+                ]" />
+        </div>
+
+        <div
+            :class="[
+                $style['c-rating-mask'],
+                $style['c-rating-stars-icons']
+            ]"
+            :style="`--starRatingPercentage: ${getRatingStarPercentage}%`">
+            <star-filled-icon
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="[
+                    $style['c-rating-star'],
+                    $style['c-rating-star--filled'],
+                    $style[`c-rating-star--${starRatingSize}`]
+                ]" />
+        </div>
+    </div>
+</template>
+
+<script>
+import {
+    StarFilledIcon,
+    StarIcon
+} from '@justeattakeaway/pie-icons-vue';
+
+export default {
+    components: {
+        StarFilledIcon,
+        StarIcon
+    },
+
+    props: {
+        starRating: {
+            type: Number,
+            default: 0
+        },
+        maxStarRating: {
+            type: Number,
+            default: 5
```

Comment:
If this can be configured, should this be a "multi-star variant" rather than specifically a five-star variant?

Reply:
Yeah. I may also change the component name too as having a flexible amount may be a good idea.

Reply:
Have updated just the name of the files now. I think i'll leave default as 5 but it can actually work with any number. Will update the readme too.
![Screenshot 2022-10-27 at 18 39 05](https://user-images.githubusercontent.com/2299779/198360399-b18981b5-6bc0-4fa1-9401-2f6e134cd893.png)